lua/orgmode/init.lua

```diff
@@ -65,6 +65,7 @@ function Org:init()
   self.clock = require('orgmode.clock'):new({
     files = self.files,
   })
+  self.clock:init()
```

Comment:
You can move this call inside the `new()` function and remove it from here.

Reply:
Done! Thanks for the review